pyvista/utilities/reader.py

```diff
@@ -1295,6 +1320,68 @@ def auto_detect_format(self):
     def auto_detect_format(self, value):
         self.reader.SetAutoDetectFormat(value)
 
+    def add_function(self, value: Union[int, Plot3DFunctionEnum]):
+        """Specify additional functions to read.
+
+        Parameters
+        ----------
+        value
+            An integer or `Plot3DFunctionEnum`.
+        """
```

Comment:
An example would be useful, in particular it won't be obvious that the user has to use `pyvista.reader.Plot3DFunctionEnum`.